<ui version="4.0" >
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="enabled" >
   <bool>true</bool>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>430</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>300</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Annchienta Editor</string>
  </property>
  <widget class="QTabWidget" name="tabWidget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>381</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Maximum" hsizetype="Maximum" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="currentIndex" >
    <number>0</number>
   </property>
   <widget class="QWidget" name="configurationTab" >
    <attribute name="title" >
     <string>Settings</string>
    </attribute>
    <widget class="QLabel" name="label" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>160</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>Game working directory:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="selectGameDirectoryDisplay" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>30</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text" >
      <string>.</string>
     </property>
    </widget>
    <widget class="QPushButton" name="selectGameDirectoryButton" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>60</y>
       <width>251</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text" >
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="gridBox" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>100</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text" >
      <string>Draw Grid</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>130</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>TileWidth:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="tileWidthBox" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>150</y>
       <width>62</width>
       <height>30</height>
      </rect>
     </property>
     <property name="value" >
      <number>64</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="tileHeightBox" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="geometry" >
      <rect>
       <x>90</x>
       <y>150</y>
       <width>62</width>
       <height>30</height>
      </rect>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
     <property name="value" >
      <number>32</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3" >
     <property name="geometry" >
      <rect>
       <x>90</x>
       <y>130</y>
       <width>77</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>TileHeight:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="mapTab" >
    <attribute name="title" >
     <string>Map</string>
    </attribute>
    <widget class="QPushButton" name="newMapButton" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text" >
      <string>New</string>
     </property>
    </widget>
    <widget class="QPushButton" name="openMapButton" >
     <property name="geometry" >
      <rect>
       <x>80</x>
       <y>10</y>
       <width>76</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text" >
      <string>Open</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveMapAsButton" >
     <property name="geometry" >
      <rect>
       <x>190</x>
       <y>10</y>
       <width>74</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text" >
      <string>Save As</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveMapButton" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="geometry" >
      <rect>
       <x>190</x>
       <y>40</y>
       <width>74</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text" >
      <string>Save</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>160</y>
       <width>251</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title" >
      <string>Layers</string>
     </property>
     <widget class="QPushButton" name="addLayerButton" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>30</y>
        <width>161</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text" >
       <string>Add layer with offset</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="addLayerZBox" >
      <property name="geometry" >
       <rect>
        <x>170</x>
        <y>30</y>
        <width>62</width>
        <height>30</height>
       </rect>
      </property>
      <property name="suffix" >
       <string/>
      </property>
      <property name="maximum" >
       <number>199</number>
      </property>
      <property name="value" >
       <number>30</number>
      </property>
     </widget>
     <widget class="QPushButton" name="nextLayerButton" >
      <property name="geometry" >
       <rect>
        <x>10</x>
        <y>80</y>
        <width>101</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text" >
       <string>Next layer</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="layerZBox" >
      <property name="geometry" >
       <rect>
        <x>170</x>
        <y>100</y>
        <width>62</width>
        <height>30</height>
       </rect>
      </property>
      <property name="suffix" >
       <string/>
      </property>
      <property name="maximum" >
       <number>199</number>
      </property>
      <property name="value" >
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_5" >
      <property name="geometry" >
       <rect>
        <x>170</x>
        <y>80</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text" >
       <string>Offset:</string>
      </property>
     </widget>
     <widget class="Line" name="line" >
      <property name="geometry" >
       <rect>
        <x>20</x>
        <y>70</y>
        <width>211</width>
        <height>3</height>
       </rect>
      </property>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSpinBox" name="layerOpacityBox" >
      <property name="geometry" >
       <rect>
        <x>170</x>
        <y>150</y>
        <width>62</width>
        <height>30</height>
       </rect>
      </property>
      <property name="maximum" >
       <number>255</number>
      </property>
      <property name="value" >
       <number>255</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_7" >
      <property name="geometry" >
       <rect>
        <x>170</x>
        <y>130</y>
        <width>62</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text" >
       <string>Opacity:</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="refreshMapButton" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>40</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text" >
      <string>Refresh</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="editTab" >
    <attribute name="title" >
     <string>Edit</string>
    </attribute>
    <widget class="QCheckBox" name="wholeTiles" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>161</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text" >
      <string>Select whole tiles</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QGroupBox" name="tileGroupBox" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>30</y>
       <width>251</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title" >
      <string>Tile Surfaces</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading</set>
     </property>
     <property name="flat" >
      <bool>false</bool>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
     <widget class="QPushButton" name="showTileSurfacesButton" >
      <property name="geometry" >
       <rect>
        <x>20</x>
        <y>30</y>
        <width>211</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text" >
       <string>Show Tiles</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="tileSideGroupBox" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>170</y>
       <width>251</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title" >
      <string>Tile Side Surfaces</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
     <widget class="QPushButton" name="showTileSideSurfacesButton" >
      <property name="geometry" >
       <rect>
        <x>20</x>
        <y>30</y>
        <width>211</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text" >
       <string>Show Tile Sides</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="tileSideOffsetBox" >
      <property name="geometry" >
       <rect>
        <x>20</x>
        <y>60</y>
        <width>62</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimum" >
       <number>-999</number>
      </property>
      <property name="maximum" >
       <number>999</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_6" >
      <property name="geometry" >
       <rect>
        <x>90</x>
        <y>70</y>
        <width>62</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text" >
       <string>Z Offset</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="obstructionGroupBox" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>260</y>
       <width>251</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title" >
      <string>Obstruction</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
     <widget class="QComboBox" name="obstructionTypeBox" >
      <property name="geometry" >
       <rect>
        <x>20</x>
        <y>30</y>
        <width>211</width>
        <height>30</height>
       </rect>
      </property>
      <property name="insertPolicy" >
       <enum>QComboBox::NoInsert</enum>
      </property>
      <item>
       <property name="text" >
        <string>DefaultObstruction</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>NoObstruction</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>FullObstruction</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QGroupBox" name="zGroupBox" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>90</y>
       <width>251</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title" >
      <string>Height of Tiles</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
     <widget class="QSpinBox" name="tileZBox" >
      <property name="geometry" >
       <rect>
        <x>20</x>
        <y>50</y>
        <width>62</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimum" >
       <number>-999</number>
      </property>
      <property name="maximum" >
       <number>999</number>
      </property>
      <property name="value" >
       <number>8</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_4" >
      <property name="geometry" >
       <rect>
        <x>20</x>
        <y>30</y>
        <width>62</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text" >
       <string>Raise by:</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="pythonTab" >
    <attribute name="title" >
     <string>Python</string>
    </attribute>
    <widget class="QPushButton" name="pythonExecuteButton" >
     <property name="geometry" >
      <rect>
       <x>140</x>
       <y>300</y>
       <width>121</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text" >
      <string>Execute</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="pythonCode" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>251</width>
       <height>281</height>
      </rect>
     </property>
     <property name="html" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:10pt; font-weight:400; font-style:normal;">
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>&lt;/body>&lt;/html></string>
     </property>
     <property name="acceptRichText" >
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pythonClearButton" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>300</y>
       <width>121</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text" >
      <string>Clear</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="mousePositionLabel" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>400</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Mouse position</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
